src/View/Dish/index.html.twig

```diff
@@ -0,0 +1,39 @@
+{% extends 'layout-visitor.html.twig' %}
+
+
+
+{% block content %}
+    <section class="menupage">
+    <h1 class="menutitle">Notre Carte</h1>
+        <div class="menupricebox">
+            <p class="dishprices">Menu Petit Tonneau Entrée/Plat ou Plat/Dessert 22 €</p>
+            <p class="dishprices">Menu Grand Tonneau: Entrée/Plat/Dessert 28 €</p>
+        </div>
+    {% for category in categories %}
+        <h2 class="menusection">{{ category.name }}  : {{ category.price }}  € <img src="/assets/images/faitmaison.png" alt="fait maison logo" class="faitmaisonlogo"></h2>
+        {% for dishesWithCategory in dishesWithCategories %}
+            <ul class="menulist">
+            {% for dish in dishesWithCategory %}
+                {% if dish.name == category.name %}
```

Comment:
non pas besoin de if, il faut que tu boucles sur `{% for dish in dishesWithCategory[category.id] %}`

Reply:
ok